packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -2,6 +2,10 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// We disable these lints, because they relate to InkResponseState
+// that is actually private, but visible for testing.
+// ignore_for_file: public_member_api_docs, library_private_types_in_public_api
```

Comment:
Please don't do this. We should try to keep that object private. 

If we can't keep it private, we will have to add docs. But We should really try hard to not make this public.

Reply:
Agreed.
We normally do not test disposal as the testing is complicated. So, I rolled back testing and will request the testing exempt. When we release the leak detector, we will be able to test cover such things.